ci/build-test-matrix.js

```diff
@@ -242,6 +244,11 @@ async function shard(configs) {
             .join(" "),
         }
       ));
+      sharded.push(Object.assign(
+        {},
+        config,
+        { bucket: '--test wast' },
+      ));
```

Comment:
Is this not going to break for the crates which support 32-bit but don't have a `tests/wast.rs`? I.e. all of them other than `wasmtime`? But even the `wasmtime` crate doesn't have that test, only `wasmtime-cli` does... I guess I am pretty confused around the logic here...

Reply:
This is abusing the fact that `cargo test --workspace --test wast` only actually executes tests named `wast` and everything else is ignored. So this effectively runs `cargo test --test wast` and nothing else is named `wast`. 

In the long-run I want to get more 32-bit tests running, just needs more support in Pulley for that.